### PR TITLE
Release/1.0.0

### PR DESCRIPTION
**Pull request by cs-ankit-gupta**
Description: Nozomi Solution Pack helps in getting all the involved Assets, Nodes, Links and CVEs from Nozomi for an Alert.
UTC: 
1. Verified that Nozomi Solution Pack get installed successfully.
2. Verified that "Nozomi Scenario" Simulation execute properly and create records.
3. Verified that On Alert Creation Playbook executes and correlate Assets, Nodes, Links and CVEs.
4. Verified that Deploy Decoy And Isolate Playbook executes succesfully.

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
+{
+    "name": "nozomi",
```
Comment (cs-suyog-jinturkar) [inline info.json]: we will need add proper name here

File: info.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    "version": "1.0.0",
+    "fsrMinCompatibility": "7.2.0",
```
Comment (cs-suyog-jinturkar) [inline info.json]: Please confirm the min version compatibility once

File: info.json [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        {
+            "name": "oT-VulnerabilityManagement",
+            "type": "solutionpack",
+            "label": "OT - Vulnerability Management",
+            "minVersion": "1.0.0"
+        }
+    ],
+    "prerequisite": null,
+    "publisher": "Fortinet",
+    "description": "Nozomi Solution Pack helps in getting all the involved Assets, Nodes, Links and CVEs from Nozomi for an Alert.",
```
Comment (cs-suyog-jinturkar) [inline info.json]: This needs to be more descriptive

File: info.json [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        "OT & IoT Security"
+    ],
+    "supportInfo": null,
+    "iconLarge": null,
+    "postInstallConfig": null,
+    "date": "2023-03-29T09:01:13+00:00",
+    "contents": {
+        "playbooks": [
+            {
+                "name": "10 - SP - Nozomi CMC"
```
Comment (cs-suyog-jinturkar) [inline info.json]: it should be usecase collection, right? why 10 - SP collection?

File: records/attachments/files/ed065a8c-aedc-4da3-b454-2ee6d5f5d507
Comment (cs-suyog-jinturkar) [inline records/attachments/files/ed065a8c-aedc-4da3-b454-2ee6d5f5d507:1]: Can we have proper name for these files?

File: playbooks/10 - SP - Nozomi CMC/> Create Asset Record.json [hunk 5]
```diff
@@ -0,0 +1,413 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "> Create Asset Record",
+    "aliasName": null,
+    "tag": null,
```
Comment (cs-suyog-jinturkar) [inline playbooks/10 - SP - Nozomi CMC/> Create Asset Record.json]: Need respective tags for all PBs, please do the needful

File: info.json (hunk 1 above)
Comment (cs-ankit-gupta) [reply info.json]: Incorporated

File: info.json (hunk 3 above)
Comment (cs-ankit-gupta) [reply info.json]: Incorporated

File: info.json (hunk 4 above)
Comment (cs-ankit-gupta) [reply info.json]: Yes it should be UseCase. Incorporated

File: records/attachments/files/ed065a8c-aedc-4da3-b454-2ee6d5f5d507
Comment (cs-ankit-gupta) [reply records/attachments/files/ed065a8c-aedc-4da3-b454-2ee6d5f5d507:1]: Incorporated

File: playbooks/10 - SP - Nozomi CMC/> Create Asset Record.json (hunk 5 above)
Comment (cs-ankit-gupta) [reply playbooks/10 - SP - Nozomi CMC/> Create Asset Record.json]: tag are there in "recordTags" filled in same json file

File: records/scenario/scenario0001.json [hunk 6]
```diff
@@ -0,0 +1,38 @@
+[
+    {
+        "@type": "Scenario",
+        "configuration": null,
+        "title": "Nozomi Scenario",
+        "description": "This Scenario adds 10 CVEs, 5 ICS Advisory, 5 KEV Alert and \"Inactive protocol OFF\" Alert. These data can be used to understand Nozomi Use Case.",
```
Comment (cs-suyog-jinturkar) [inline records/scenario/scenario0001.json:6]: Alerts

File: records/scenario/scenario0001.json [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+        "steps": {
+            "Steps": [
+                {
+                    "name": "Alert Creation Scenario",
+                    "type": "executePlaybook",
+                    "playbookIRI": "\/api\/3\/workflows\/f9bdb309-cc12-461f-9613-0d9d08f11406",
+                    "preCreateMessage": "Creating Alert and Asset demo record for Stuxnet Alert",
```
Comment (cs-suyog-jinturkar) [inline records/scenario/scenario0001.json]: record or records?

File: records/scenario/scenario0001.json (hunk 6 above)
Comment (cs-ankit-gupta) [reply records/scenario/scenario0001.json:6]: Scenario contain only one Alert, i.e., it is Alert

File: records/scenario/scenario0001.json (hunk 7 above)
Comment (cs-ankit-gupta) [reply records/scenario/scenario0001.json]: incorporated

File: info.json (hunk 2 above)
Comment (cs-ankit-gupta) [reply info.json]: Changed to 7.3.0

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**